AdminCore.ScheduledJobs/SchedulingManager.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AdminCore.ScheduledJobs
+{
+  public class SchedulingManager
+  {
+    private static void Main()
+    {
+      //TODO remove unused main method
```

Comment:
What is the best way to handle this? Unused main method but project needs it to compile